File: DevExpress VCL 23.2.6 Full Source/VCL/ExpressGantt Control/Demos/Delphi/FeaturesDemo/FeaturesDemoMain.pas
unit FeaturesDemoMain;

interface

{$I cxVer.inc}

uses
  Windows, Messages, SysUtils, Classes, Graphics, Controls, Forms,
  Dialogs, Menus, ComCtrls, StdCtrls,
  dxCore, cxContainer, cxEdit, cxTextEdit, cxDropDownEdit, cxMaskEdit, cxSpinEdit,
  cxGraphics, cxClasses, cxControls,
{$IFDEF EXPRESSSKINS}
  dxSkinsdxRibbonPainter,
  dxSkinsdxBarPainter,
{$ENDIF}
  cxLookAndFeels, cxLookAndFeelPainters, dxLayoutControl,
  dxLayoutContainer, dxLayoutLookAndFeels, dxLayoutcxEditAdapters, DemoBasicMain,
  dxGanttControlCustomClasses, dxGanttControl, dxGanttControlCustomSheet, dxGanttControlViewChart,
  dxGanttControlViewResourceSheet, dxGanttControlViewTimeline, dxGanttControlTasks, dxGanttControlAssignments,
  dxGanttControlResources;

type
  TFeaturesDemoMainForm = class(TDemoBasicMainForm)
  public
    constructor Create(AOwner: TComponent); override;
  end;

var
  FeaturesDemoMainForm: TFeaturesDemoMainForm;

implementation

{$R *.dfm}

constructor TFeaturesDemoMainForm.Create(AOwner: TComponent);
begin
  inherited Create(AOwner);
  GanttControl.DataModel.LoadFromFile('..\..\Data\SoftDev.xml');
end;

end.

File: DevExpress VCL 23.2.6 Full Source/VCL/ExpressGantt Control/Demos/CBuilder/FeaturesDemo/FeaturesDemoMain.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "FeaturesDemoMain.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "cxClasses"
#pragma link "cxContainer"
#pragma link "cxControls"
#pragma link "cxDropDownEdit"
#pragma link "cxEdit"
#pragma link "cxGraphics"
#pragma link "cxLookAndFeelPainters"
#pragma link "cxLookAndFeels"
#pragma link "cxMaskEdit"
#pragma link "cxSpinEdit"
#pragma link "cxTextEdit"
#pragma link "dxCore"
#pragma link "dxGanttControl"
#pragma link "dxGanttControlAssignments"
#pragma link "dxGanttControlCustomClasses"
#pragma link "dxGanttControlCustomSheet"
#pragma link "dxGanttControlResources"
#pragma link "dxGanttControlTasks"
#pragma link "dxGanttControlViewChart"
#pragma link "dxGanttControlViewResourceSheet"
#pragma link "dxGanttControlViewTimeline"
#pragma link "dxLayoutContainer"
#pragma link "dxLayoutControl"
#pragma link "dxLayoutcxEditAdapters"
#pragma link "dxLayoutLookAndFeels"
#pragma resource "*.dfm"
TFeaturesDemoMainForm *FeaturesDemoMainForm;
//---------------------------------------------------------------------------
__fastcall TFeaturesDemoMainForm::TFeaturesDemoMainForm(TComponent* Owner)
	: TDemoBasicMainForm(Owner)
{
  GanttControl->DataModel->LoadFromFile("..\\..\\Data\\SoftDev.xml");
}
//---------------------------------------------------------------------------


